src/ClientData/FunctionInfo.cpp

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "ClientData/FunctionInfo.h"
+
+#include <absl/strings/match.h>
+#include <absl/strings/str_join.h>
+#include <xxhash.h>
+
+#include "ClientData/ModuleData.h"
+#include "ClientData/ProcessData.h"
+#include "ObjectUtils/Address.h"
+
+namespace orbit_client_data {
+
+namespace {
+uint64_t StringHash(const std::string& string) {
+  return XXH64(string.data(), string.size(), 0xBADDCAFEDEAD10CC);
+}
+}  // namespace
+
+std::string FunctionInfo::GetLoadedModuleName() const {
+  return ModuleData::GetLoadedModuleNameByPath(module_path());
+}
```

Comment:
1. I think we should remove the "Loaded" from both of these function names
2. This is not a lot, so I would actually to prefer to remove `ModuleData::GetLoadedModuleNameByPath` and `FunctionInfo::GetLoadedModuleName()` completely and just use `std::filesystem::path{...}.filename()` directly. Especially the static ModuleData method feels weird, since ModuleData already has a `name()` method. (btw `name()` is different to the filename of filepath, which is a whole different problem b/222283082). Alternatively maybe name the FunctionInfo method: `GetModuleFilename()` to make clear its the filename, not the "real" name